doc/md_src_files/060_decision-trees.md

```diff
@@ -101,13 +101,54 @@ Once the decision points are selected and the prioritization labels agreed upon,
+In our case, we are not attempting to fit a tree to data.
+Rather, we are interested in producing usable trees that minimize extraneous effort.
+To that end, a brief examination of what qualities decision tree construction  methods optimize for is warranted.
+
+Decision tree construction methods must address four significant concerns: feature selection, feature type, overfitting, and parsimony.
+
+Feature selection is perhaps the most important consideration for SSVC, because it directly affects the information gathering requirements placed on the analyst attempting to use the tree.
+Each decision point in SSVC is a feature.
+
 The SSVC version 1  ~applier~ deployer tree had 225 rows when we wrote it out in long text form.
 It only has four outcomes to differentiate between.
 Thus, on average that decision process treats one situation (combination of decision values) as equivalent to 65 other situations.
 If nothing else, this means analysts are spending time gathering evidence to make fine distinctions that are not used in the final decision.
 The added details also make it harder for the decision process to accurately manage the risks in question.
 This difficulty arises because more variance and complexity there is in the decision increases the possibility of errors in the decision process itself.
 
+Different feature types (nominal, ordinal, interval, ratio) require slightly different approaches when building decision trees.
+Decision trees work best on features that can be ordered (i.e., where the greater-than or less-than operators make sense).
+Ordinal, interval, and ratio feature types can be ordered.
+All of the features included in SSVC v2 can be considered ordinal data. 
+That is, while they can be ordered (e.g., for Exploitation, active is greater than poc is greater than none), they can not be compared via subtraction or division (active - poc = nonsense).
+
+Those wishing to customize SSVC should take care when adding a nominal type feature though.
+Nominal features cannot be sorted and might be better suited to being  converted to binary variables (i.e., one-hot encoding) before further analysis of the sort we are discussing here.
+
+Overfitting increases tree complexity by completely incorporating all the noise in the training data set into the decision points in a tree. 
+In our case, our "data" is just the set of outcomes as decided by humans, so overfitting is less of a concern, assuming the feature selection has been done with care.
+
+Parsimony is, in essence, Occam's Razor applied to tree selection: Given the choice between two trees that have identical outputs, choose the one with fewer decisions. 
+When applied to the induction algorithms used to fit a decision tree to a training data set, this often takes the form of ordering decision points in descending order based on the amount of information gained at each step.
```

Comment:
I think this is out of place, unless you qualify that this is the measure used in machine learning and we may borrow it by analogy but it is not what we are doing here.

Reply:
I just removed the ordering sentence from 6fbb0de